diff --git a/deps/CMakeLists.txt b/deps/CMakeLists.txt
index a8bf132..65474e6 100644
--- a/deps/CMakeLists.txt
+++ b/deps/CMakeLists.txt
@@ -66,6 +66,7 @@ endif()
 
 # Libraries are built for Mac/Windows, and consumed via the Package Manager for Linux
 if (NOT UNIX OR APPLE OR DEFINED USE_SUBMODULE_SOURCES) # Windows and Mac
+    set(FT_DISABLE_HARFBUZZ TRUE CACHE BOOL "" FORCE)
     add_subdirectory(${freetype_SUBMOD_DIR})
     set_target_properties(freetype PROPERTIES DEBUG_POSTFIX "")
     set_target_properties(freetype PROPERTIES ARCHIVE_OUTPUT_DIRECTORY ${OutputDirectory})

